docs/book/src/topics/accessing-ec2-instances.md

```diff
@@ -31,6 +31,7 @@ spec:
   bastion:
     enabled: true
 ```
+If this field is set, then by default the latest AMI(Ubuntu 20.04 LTS OS) is looked up from [Ubuntu cloud images](https://ubuntu.com/server/docs/cloud-images/amazon-ec2) by CAPA controller and used in bastion host creation.
```

Comment:
@sedefsavas Let me know if this statement would suffice?

Reply:
Just added `this is true in case AMI ID is not specified`.